x-pack/plugins/code/public/components/main/main.scss

```diff
@@ -272,6 +272,10 @@
   min-width: unset !important;
 }
 
+.code-no-max-width {
+  max-width: unset !important;
```

Comment:
The max-width on breadcrumbs is extremely important to ensure that the items get truncated leaving room for the most current (last) breadcrumb. It's the only way to maintain consistent layout from app to app at all screen sizes. 

That being said, it should **not** be truncating the last breadcrumb unless it reaches the far end of the header.

<img width="965" alt="Screen Shot 2019-04-01 at 17 32 19 PM" src="https://user-images.githubusercontent.com/549577/55361088-209c6180-54a4-11e9-8485-61fc2fe12a7b.png">

By glancing at your screenshot, it looks like the last breadcrumb "exists" but is blank. If you make sure not to show a blank last breadcrumb, you won't need to override the max-width.

<img src="https://d.pr/free/i/sArDAG+" width="50%" />

Reply:
The last breadcrumb "exists" but is blank is because in the second last breadcrumb, we want to add a link to this project's root url, but if the breadcrumb isn't the last, the `href` props of the breadcrumb is invalid, so I add a blank breadcrumb to make it second last.

Reply:
@cchaos so ideally we want the last component of breadcrumb also clickable, this is more of a workaround, do you think it make sense to do that on the EUI side?

Reply:
Yes, you should make an issue in EUI to allow support for href's on the last breadcrumb. Another problem with this current solution is that it can also be misleading and the user think that there should be more breadcrumbs since there is a trailing slash.

Reply:
@cchaos this is actually a design decision we made. We want to keep the last `/` to concatenate with the file path component below. So that visually, they kinda look as a whole.

![image](https://user-images.githubusercontent.com/987855/55527477-ce3d8980-56cb-11e9-95b1-121c9932f9e1.png)

Reply:
I think it's fine to have the trailing slash in your app's content as shown by the screenshot. However, the Kibana header is used for navigation for all apps and we should stick to a pattern that is consistent. That pattern being, that there are no trailing slashes and the last breadcrumb indicates your current place in the app (not a file tree).

Again, the original concern with your CSS override is that it may cause issues with truncating when necessary (on small screens and if that breadcrumb is too long to fit in the header space). The easiest way to combat this is to just use the default patterns provided by the breadcrumbs and just use the last breadcrumb as your current location.